src/main/java/jenkins/plugins/instana/HttpRequestExecution.java

```diff
@@ -48,8 +49,26 @@ static HttpRequestExecution from(ReleaseMarker http, TaskListener taskListener)
 		JSONObject jsonObject =  new JSONObject();
 		jsonObject.put("name", http.getReleaseName());
 		jsonObject.put("start", http.getReleaseStartTimestamp());
+		if (http.getServiceNames() != null) {
+			List<JSONObject> services = new ArrayList<>();
+			for (String serviceName : http.getServiceNames()) {
+				JSONObject service = new JSONObject();
+				service.put("name", serviceName);
+				services.add(service);
+			}
+			jsonObject.put("services", services);
+		}
+		if (http.getApplicationNames() != null) {
+			List<JSONObject> applications = new ArrayList<>();
+			for (String applicationName : http.getApplicationNames()) {
+				JSONObject service = new JSONObject();
```

Comment:
Nitpick: `application`

Reply:
Good catch from copy/pasting. Changed to `application`.